src/containers/DataTable/index.js

```diff
@@ -59,6 +75,14 @@ export function DataTable ({
 					</Link>
 				</div>
 				}
+				{onDownload &&
+					<div className="float-right pr-3">
+						<Button tag="span" size="sm" onClick={downloadHandler} >
+							<i className="cil-arrow-bottom"></i>
+							Download
```

Comment:
@titkov49 Can we implement language localisations to Data table? I mean e.g. in the next iteration/sprint?

Reply:
Yep, I have it in current sprint  :)